### PR TITLE
Inheritance wallet - Part 3 - Cosigner onboarding & key replacement

### PR DESCRIPTION
**Pull request by rabbitholiness**
This PR builds on the previous inheritance wallet PRs:
[Part 1: Use case & wallet creation](https://github.com/BitcoinDesign/Guide/pull/1081)
[Part 2: Wallet backup](https://github.com/BitcoinDesign/Guide/pull/1087)

It adds a page about how [Alice is set up as a cosigner](https://deploy-preview-1091--bitcoin-design-site.netlify.app/guide/inheritance-wallet/onboarding-cosigners/) for the family savings wallet. It also adds a page about [how making changes work](https://deploy-preview-1091--bitcoin-design-site.netlify.app/guide/inheritance-wallet/making-changes/), like key replacement or editing timelocks.

### CODE REVIEW

**Comment by rabbitholiness** [comment]
Opened this PR in draft mode so far, so that people can look at it. There's quite a few things to tweak:
- Image captions & alt-texts
- Consistency checks
- Navigation & reference links
- Potentially additional clickable prototypes

**Comment by rabbitholiness** [comment]
Opening this up for initial feedback, even if there are two things left to do:
- Write alt-texts for the 66 images in this PR 😊
- Create a visual for the ["how it works" section](https://deploy-preview-1091--bitcoin-design-site.netlify.app/guide/inheritance-wallet/making-changes/#how-it-works) of the "Making changes" page, for better clarity.

File: guide/inheritance-wallet/onboarding-cosigners.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+So far, the wallet has been created on Bob's app. The next step is to get Alice set up as a cosigner and to enable her to sign transactions. To do this Alice needs to:
+
+1. Import the wallet configuration to her app.
+2. Enable her Bitbox to sign transactions.
```
[the remark this answers is not in the document]
Comment (rabbitholiness) [reply guide/inheritance-wallet/onboarding-cosigners.md:102]: I see your point. However, I would probably keep the device names. I know they're brands, but I think it's easier for the reader to follow along, if they are specific examples. In addition, these brands have been around for a long time, so I wouldn't be too worried about that.

[the remark this answers is not in the document]
Comment (GBKS) [reply guide/inheritance-wallet/onboarding-cosigners.md:102]: How about we make up our brands - BitGuard, CoinSafe, and BlockVault? I've done this before with mock-ups and it usually works well.

[the remark this answers is not in the document]
Comment (GBKS) [reply guide/inheritance-wallet/onboarding-cosigners.md:102]: Example:

image

File: guide/inheritance-wallet/making-changes.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+
+Bob taps on the reminder and starts creating the transaction. Because the app knows about both wallets, it will prefill most of the information in the send flow, like the recipient address and the transaction amount. 
+
+{% include image-gallery.html pages = page.images_fund-transfer-bob %} 
+
+After creating the transaction, Bob directly continues to sign it with his Trezor. Once this is done, Alice will provide her signature.
+
+#### Alice co-signs the transaction
+
+On the app home screen, Alice now sees a reminder that a transaction is ready to be signed. On the detail screen, she also sees that Bob has already provided a signature. Alice hits "sign" and goes through the transaction signing flow on her Bitbox.
```
Comment (GBKS) [inline guide/inheritance-wallet/making-changes.md:254]: > now sees a reminder that a transaction is ready to be signed

How does the app know about the transaction? Is there a server handling communication? If not, then Bob would have to share the PSBT.

File: guide/inheritance-wallet/making-changes.md [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+
+#### Alice co-signs the transaction
+
+On the app home screen, Alice now sees a reminder that a transaction is ready to be signed. On the detail screen, she also sees that Bob has already provided a signature. Alice hits "sign" and goes through the transaction signing flow on her Bitbox. 
+
+{% include image-gallery.html pages = page.images_alice-sign-transfer %} 
+
+After she has signed the transaction, she sees that the transaction is being confirmed by the network. Once the transaction has been confirmed, the app home screen shows a reminder that the original wallet should be archived. 
+
+### Archiving the old wallet
```
Comment (GBKS) [inline guide/inheritance-wallet/making-changes.md:260]: Could cross-link to the [archival paragraph](https://deploy-preview-1091--bitcoin-design-site.netlify.app/guide/multiple-wallets/#archival) in the multi-wallet page somewhere.

**Review by GBKS — CHANGES_REQUESTED** [review]
Looking good. Just some minor feedback.

**Comment by rabbitholiness** [comment]
I will open a new PR, as I had technical issues with the existing branch.

**Pull request closed without merging**